### PR TITLE
Fix selection appearing after closing char

### PR DESCRIPTION
**Pull request by maximbaz**
@alexherbo2 this fixes https://github.com/alexherbo2/auto-pairs.kak/issues/2#issuecomment-402624901

### CODE REVIEW

**Comment by maximbaz** [comment]
Not sure if this now closes #3 or you want to keep that one open.

**Comment by alexherbo2** [comment]
It doesn’t honour <kbd>a</kbd> command.

**Comment by maximbaz** [comment]
I'm not sure what you mean...

**Comment by alexherbo2** [comment]
The selections are not preserved after using the <kbd>h</kbd> command (in your fix).

**Comment by maximbaz** [comment]
Hey, I've just logged into irc and seen your messages! So, was the spiricious selection always an issue?

**Comment by alexherbo2** [comment]
Yep.

By inserting an opener pair with say the <kbd>a</kbd> command, you can see that the selection gets shifted.

File: rc/auto-pairs.kak [hunk 1]
```diff
@@ -8,9 +8,14 @@ define-command -hidden -params 2 auto-pairs-insert-opener %{ try %{
     if [ "$1" = "$2" ]; then
       echo execute-keys -draft '2h<a-K>\w<ret>'
     fi
+    echo execute-keys -draft '";<a-K>\w<ret>"'
+    echo execute-keys -no-hooks '"%arg{2}<a-;>"'
+    if echo "$kak_selections_desc" | grep -Eq '^(([0-9]+)\.([0-9]+),\2\.\3:?)+$'; then
```
Comment (maximbaz) [inline rc/auto-pairs.kak]: in the most recent kakoune the delimeter has changed from `:` to a space, just FYI...

**Comment by maximbaz** [comment]
@alexherbo2 I don't think the builds are entirely fixed yet 😛 

```
g++: error: unrecognized command line option ‘-std=gnu++14’
make: *** [.alias_registry.debug.o] Error 1
0.01s$ export PATH=$PWD/bin:$PATH
cd test
./run
/home/travis/.travis/job_stages: line 79: cd: test: No such file or directory
/home/travis/.travis/job_stages: line 80: ./run: No such file or directory
The command "export PATH=$PWD/bin:$PATH
cd test
./run
" exited with 127.
```

**Pull request closed without merging**